content/20.treatments.md

```diff
@@ -166,6 +166,82 @@ A list of current studies and their results, using carefully the information req
 
 Summarize the state of the antiviral approach.
 
+##### Hydroxychloroquine and Chloroquine
+
+Hydroxychloroquine (HCQ) and chloroquine (CQ) are quinolones that have been widely used as anti-malarial and disease-modifying anti-rheumatic drugs in humans.
+HCQ and CQ act through the same mechanism of action, but HCQ has a better safety profile [@doi:10.1136/ard.2008.101766].
+HCQ/CQ have been reported to have anti-viral effects, including in HIV-1, SARS-CoV, MERS-CoV, and SARS-CoV-2 [@doi:10.1016/S1473-3099(06)70361-9; @doi:10.1128/AAC.03011-14; @doi:10.1093/cid/ciaa237; @doi:10.1038/s41422-020-0282-0].
```

Comment:
```suggestion
HCQ/CQ have been reported to have anti-viral effects, including in HIV-1, SARS-CoV, MERS-CoV, and SARS-CoV-2 [@doi:10/dh8phb; @doi:10.1128/AAC.03011-14; @doi:10.1093/cid/ciaa237; @doi:10.1038/s41422-020-0282-0].
```
DOIs with parentheses are a little tricky -- the reference manager in Manubot (that generates the manuscript) doesn't like them! I plugged this one into shortdoi.org to get a Manubot-friendly version